user-docs/pupil-capture.md

```diff
@@ -72,6 +73,8 @@ Adjust the headset according to your comfort and to ensure a good calibration re
 
 <!-- First select the calibration method you would like to use: -->
 
+Before starting the calibration, make sure to secure the headset comfortably to ensure a good calibration result.
```

Comment:
Before starting calibration, ensure that eye(s) are robustly detected and that the headset is comfortable for the participant.